s3/development/sc-s3-encrypted-ra-development.yaml

```diff
@@ -73,6 +73,13 @@ Resources:
             Condition:
               'Null':
                 s3:x-amz-server-side-encryption: 'true'
+  S3BucketTagger:
+    Type: Custom::S3BucketTagger
+    DependsOn: S3Bucket
```

Comment:
i don't think you need this since there's already a !Ref on `BucketName`

Reply:
removed